ContactsManager/ContactsManager_select_method.html

```diff
@@ -0,0 +1,98 @@
+<div>{{draft}}{{securecontext_header}}{{DefaultAPISidebar("Contact Picker API")}}</div>
+
+<p class="summary">The <strong><code>select()</code></strong> method of the {{domxref("ContactsManager")}} interface returns a {{jsxref('Promise')}} which, when resolved, presents the user with a contact picker which allows them to select contact(s) they wish to share.</p>
+
+<h2 id="Syntax">Syntax</h2>
+
+<pre class="syntaxbox notranslate">var <var>ContactInfo</var> = ContactsManager.select(properties, options);</pre>
+
+<h3 id="Parameters">Parameters</h3>
+
+<dl>
+  <dt><em>properties</em></dt>
+  <dd>An array of {{jsxref('String', 'strings')}} defining what information to retrieve from a contact. Allowed values are as follows:
+    <!-- I'm unsure of this because in reality it's whatever in contained in each of these fields, not the actually name/email etc... -->
```

Comment:
I'm unsure if we need to point that out. It should be obvious. On the other hand, it could also be something that no one thinks about until something fails because the wrong data was entered.